doc: add change doc to specify max plaintext length input on encrypt (#138)
awslabs/aws-encryption-sdk-specification: client-apis/encrypt.md

The [frame length](../data-format/message-header.md#frame-length) to use for [framed data](../data-format/message-body.md).
This value MUST NOT exceed the value 2^32 - 1.

### Plaintext Length Bound

> A bound on the length of the [plaintext](#plaintext) to encrypt.

Reviewer: ```suggestion
A bound on the length of [plaintext](#plaintext) with an unknown length to encrypt.
```
I think we should be clear here that this value is for.

  Otherwise, this is an empty encryption context.
- Algorithm Suite: If provided, this is the [input algorithm suite](#algorithm-suite).
  Otherwise, this field is not included.

> - Max Plaintext Length: If the [input plaintext](#plaintext) has known length, this is the
>   length of the plaintext.

Reviewer: ```suggestion
- Max Plaintext Length: If the [input plaintext](#plaintext) has known length,
  this length MUST be used.
```

  - The plaintext contains part of the input [plaintext](#plaintext) this frame is encrypting.
- [Authentication Tag](../data-format/message-body.md#authentication-tag): MUST be the authentication tag outputted by the above encryption.

If [Plaintext Length Bound](#plaintext-length-bound) was specified on input

> and this operation can determine that the plaintext being encrypted has a length greater than this value,

Reviewer: ```suggestion
and this operation determines that the plaintext being encrypted has a length greater than this value,
```
How could the operation not determine this?
This also resolves my "top billing comment" :)
